msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/SilentRequestHelper.java

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.aad.msal4j;
+
+import java.util.Date;
+import org.slf4j.Logger;
+
+class SilentRequestHelper {
+
+    private static final int ACCESS_TOKEN_EXPIRE_BUFFER_IN_SEC = 5 * 60;
+
+    private SilentRequestHelper() {
+        // Utility class
+    }
+
+    static CacheRefreshReason NeedsRefresh(SilentParameters parameters, AuthenticationResult cachedResult, Logger log) {
```

Comment:
nit: Method names generally start with a lower-case letter in Java: https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html

Also, since it's returning a `CacheRefreshReason` instead of a true/false, something like `getRefreshReason` or `determineRefreshReason` might be a better name.

Reply:
Updated